Assets/World/Units/Unit.gd

```diff
@@ -35,6 +35,7 @@ const DIRECTION = [
 # Generic properties
 export var unit_name = "Untitled" # user defined name for the unit
 export(Global.Faction) var faction := 0 setget set_faction
+var health = -1	#this is so the default health must be set but also won't auto destroy itself if not set
```

Comment:
```suggestion
var health = -1 # health must be set or it won't auto destroy itself
```
I shorted the comment a bit. Also it must follow the format EXPRESSION SPACE HASHTAG COMMENT. No tab before the hashtag.